.changeset/cool-walls-applaud.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/catalog-model': minor
+'@backstage/plugin-api-docs': minor
```

Comment:
```suggestion
'@backstage/catalog-model': patch
'@backstage/plugin-api-docs': patch
```

These should just be patch as they are just updates to documentation, you might not even need the one for `@backstage/catalog-model` but I'll let the Maintainers make the call on that 👍